source/includes/steps-install-shell-macos.yaml

```diff
@@ -23,5 +23,17 @@ source:
   file: steps-install-shell-base.yaml
   ref: add-shell-to-path
 ref: add-shell-to-path-macos
+---
+title: "Allow macOS to Run ``mongosh``."
+ref: remove-quarantine-macos
+level: 4
+content: |
+   macOS may prevent ``mongosh`` from running after installation. If
+   you receive a security error when starting ``mongosh`` indicating
+   that the developer could not be identified or verified, run
+   the following command:
+
+   .. code-block:: sh
 
+      xattr -r -d com.apple.quarantine /path/to/mongosh
```

Comment:
- Careful with this. Apple changes how it quarantines executables every version of macOS since it started doing this in 10.13. This command here works for me on 10.14, but I wonder about 10.13, and the future 10.16 (I'm assuming it worked for you on your 10.15 workstation).

- I use this copy for the install docs:
https://github.com/mongodb/docs/blob/8dc743535997ab1e68ba459f0ad2a18fe631f7a1/source/tutorial/install-mongodb-on-os-x.txt#L170
It's big, bulky, and I have to include it twice (once for `mongod` once for `mongo`), but it's GUI, familiar, and the same between versions. I don't want you to use this whole bulky copy from the install doc, it really is ugly & I hate it, but maybe something much shorter.

- the `-r` flag makes the command recursive, according to the man page. There's nothing to recurse under the `mongosh` executable itself. However, removing that flag made the command error for me, returning it restored functionality. ??

- can you run this command as a non-administrator? Internet search inconclusive.

Reply:
- I saw your copy used on the main install docs. I was trying to avoid the bulky copy but you bring up a good point about this differing from version to version. Despite it being bulky, I can't think of a much clearer way to outline these actions. Still waiting on single-sourcing across multiple repos 🙃 

- Re `-r` flag: Weird right?? @mmarcon not sure if you know anything about this behavior?

- @mmarcon dropped a JIRA comment stating:

  > Note that sudo is only required for xattr if the binary is saved in a location where the user has no permissions. For example, I have it my ~/.bin folder and I could run the command without sudo.

Reply:
Good sleuthing. I think the copy you used to walk users through this process is perfect. And a bit less bulkier! Nice work.